features/admin.session-management.v1/package.json

```
@@ -22,6 +22,7 @@
         "@oxygen-ui/react-icons": "^2.4.6",
         "@wso2is/access-control": "^3.3.20",
         "@wso2is/admin.core.v1": "^2.46.30",
+        "@wso2is/admin.organizations.v1": "^2.26.211",
```

Let's use workspace protocol.

```suggestion
        "@wso2is/admin.organizations.v1": "workspace:^",
```